sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -177,7 +177,7 @@ private[ui] class RunningExecutionTable(
     showFailedJobs = true) {
 
   override protected def header: Seq[String] =
-    baseHeader ++ Seq("Running Jobs", "Succeeded Jobs", "Failed Jobs")
+    baseHeader ++ Seq("Running Job Id", "Succeeded Job Id", "Failed Jobs Id")
```

Comment:
OK, but these should be "Job IDs"

Reply:
I've changed it this way. But I don't understand why is IDs, not ID?There are no multiple JOB common execution plans.